### PR TITLE
added react-native-router-flux. set up basic front end routing with f…

### PR DESCRIPTION
**Pull request by Aberman12**
…irst couple screens.

### CODE REVIEW

File: src/components/Counter.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from "react";
 import { View, Alert } from "react-native";
+import { Card, CardSection } from "./common";
```
Comment (paulolramos) [inline src/components/Counter.js]: Is this another UI library? I already started to use native-base

**Comment by paulolramos** [comment]
I think there too many things going on in this PR. Would you be down to break it up into different issues?

File: src/components/Counter.js (hunk 1 above)
Comment (Aberman12) [reply src/components/Counter.js]: thats a components folder i had alread made. im just using some premade components from their cus it makes the building process faster for me. when were done with func we can go back and restyle everything. You can use native-base or borrow from this folder if you want

**Comment by Aberman12** [comment]
yeah i forgot to pull after i finished the first few things. what i have here is just the basic counter and then the advanced counter setup. as we work to build out the advnaced counter the basic one will automatically complete itself. For the advanced its just body weight and then we add an option to include the type of drink each time they add a new one, that way we can determine their blood alcohol level as they go through the session. all thats left after that is to setup a background timer, notifications, and a timestamp to keep track of when they enter each drink. i think this is what we had written up on the whiteboard before

**Comment by paulolramos** [comment]
I put everything that was on the whiteboard on the [projects tab](https://github.com/paulolramos/swig/projects). The plan was to complete the standard session first. There are already [issues](https://github.com/paulolramos/swig/issues) that we had agreed to tackle first.

**Comment by paulolramos** [comment]
Like we talked, I'm going to close this pull request and submit another PR just for the routing.

**Pull request closed without merging**